docs/reference/commandline/network_create.md

```diff
@@ -14,18 +14,111 @@ parent = "smn_cli"
+bridge network. You cannot remove this default bridge network but you can create
+new ones using the `network create` command.
+
+```bash
+$ docker network create -d bridge my-bridge-network
+```
+
+Bridge networks are isolated networks on a single Engine installation. If you
+want to create a network that spans multiple Docker hosts each running an
+Engine, you must create an `overlay` network. Unlike `bridge` networks overlay
+networks require some pre-existing conditions before you can create one. These
+conditions are:
+
+* Access to a key-value store. Engine supports Consul, Etcd, Zookeeper (Distributed store), and BoltDB (Local store) key-value stores.
+* A cluster of hosts with connectivity to the key-value store.
+* A properly configured Engine `daemon` on each host in the cluster.
+
+The `docker daemon` options that support the `overlay` network are:
+
+* `--cluster-store`
+* `--cluster-store-opt`
+* `--cluster-advertise`
+
+It is also a good idea, though not required, that you install Docker Swarm on
+to manage the cluster. Swarm provides sophisticated discovery and server
+management that can assist your implementation. Once you have prepared the
+`overlay` network prerequisites you simply choose a Docker host in the cluster
+and issue the following to create the network:
+
+```bash
+$ docker network create -d overlay my-multihost-network
 ```
-  $ docker network create -d overlay multi-host-network
-  $ docker run -itd --net=multi-host-network busybox
+
+Network names must be unique. The Docker daemon attempts to identify naming
+conflicts but this is not guaranteed. It is the user's responsibility to avoid
+name conflicts.
+
+## Connect containers
+
+When you start a container use the `--net` flag to connect it to a network.
+This adds the `busybox` container to the `mynet` network.
+
+```bash
+$ docker run -itd --net=my-multihost-network busybox
 ```
 
-the container will be connected to the network that is created and managed by the driver (multi-host overlay driver in the above example) or external network plugins.
+If you want to add a container to a network after the container is already
+running use the `docker network connect` subcommand.
+
+You can connect multiple containers to the same network. Once connected, the
+containers can communicate using only another container's IP address or name.
+For `overlay` networks or custom plugins that support multi-host connectivity,
+containers connected to the same multi-host network but launched from different
+Engines can also communicate in this way.
+
+You can disconnect a container from a network using the `docker network
+disconnect` command.
+
+## Specifying advanced options
+
+When you create a network, Engine creates a non-overlapping subnetwork for the network by default. You can override this default and specify a subnetwork directly using the the `--subnet` option. On a `bridge` network you can only create a single subnet:
```

Comment:
`subnet` is much more easy to understand for `advanced` users than `subnetwork`. `subnetwork` seems to indicate sub-division of an existing network. But in reality, it is a `subnet` purely for ip-addressing purposes.